op-node/node/tracer/comms_test.go

```diff
@@ -37,11 +36,6 @@ func TestTracer(t *testing.T) {
 	d := NewTracerDeriver(tr)
 	rng := rand.New(rand.NewSource(123))
 
-	l1Head := testutils.RandomBlockRef(rng)
-	d.OnEvent(context.Background(), status.L1UnsafeEvent{L1Unsafe: l1Head})
```

Comment:
Do we need to replace this assertion?

Reply:
This has to be removed, not replaced since the `TracerDeriver`'s onEvent handler does not handle the `L1UnsafeEvent`, but rather the `OnNewL1Head` method is called explicitly. 

We may still call `OnNewL1Head` by `testTracer`, but it would be not testing anything but to call dummy method works.